### DISCUSSION TITLE
Support Request: Can anyone pls send me the download of open tablet driver version that is compatible with mac os x 10.11.6 i really need it and i am in a hurry

### DISCUSSION THREAD

**Discussion by RiviDesign** [OpenTabletDriver/OpenTabletDriver#2646] [Q&A] [answered]
### Acknowledgements

- [X] I have installed OpenTabletDriver by following its [official installation instructions](https://opentabletdriver.net/).
- [X] I have checked the [OpenTabletDriver Wiki](https://opentabletdriver.net/Wiki) and respective FAQ pages, and my issue was not covered or actually fixed.
- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.

### What do you need help with?

I have downloaded the open tablet driver's latest version and it is not compatible with the mac os 10.11.6 version that is why I request that the developers have an old version of the driver

### What have you tried?

I have searched on youtube and on the GitHub page but have not been able to find any information

### Diagnostic Information

I can not the do the diagnostics as the application isn't running

**Comment by jamesbt365** [comment, marked as answer]
If that version doesnt have dotnet 6 it isnt possible. You should update your system
